media-libs/opencolorio/opencolorio-2.2.1.ebuild

```diff
@@ -0,0 +1,130 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..11} )
+
+inherit cmake python-single-r1 virtualx
+
+DESCRIPTION="A color management framework for visual effects and animation"
+HOMEPAGE="https://opencolorio.org https://github.com/AcademySoftwareFoundation/OpenColorIO"
+SRC_URI="https://github.com/AcademySoftwareFoundation/OpenColorIO/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/OpenColorIO-${PV}"
+
+LICENSE="BSD"
+SLOT="0/$(ver_cut 1-2)"
+# minizip-ng: ~arm ~arm64 ~ppc64 ~riscv
+# osl: ~riscv
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~riscv ~x86"
+IUSE="apps cpu_flags_x86_sse2 doc opengl python static-libs test"
+# TODO: drop opengl? It does nothing without building either the apps or the testsuite
+REQUIRED_USE="
+	apps? ( opengl )
+	doc? ( python )
+	python? ( ${PYTHON_REQUIRED_USE} )
+	test? ( opengl )
+"
+
+RDEPEND="
+	dev-cpp/pystring
+	>=dev-cpp/yaml-cpp-0.7.0:=
+	dev-libs/expat
+	>=dev-libs/imath-3.1.5:=
+	sys-libs/minizip-ng
+	sys-libs/zlib
+	apps? (
+		media-libs/lcms:2
+		>=media-libs/openexr-3.1.5:=
+	)
+	opengl? (
+		media-libs/freeglut
+		media-libs/glew:=
+		media-libs/libglvnd
+	)
+	python? (
+		${PYTHON_DEPS}
+		$(python_gen_cond_dep 'dev-python/pybind11[${PYTHON_USEDEP}]')
+	)
+"
+DEPEND="${RDEPEND}"
+# TODO: OSL tests would need OIIO, leading to a circular dependency. If OIIO
+# isn't found this test will be skipped (automagic if found?)
+BDEPEND="
+	virtual/pkgconfig
+	doc? (
+		app-doc/doxygen
+		$(python_gen_cond_dep '
+			dev-python/breathe[${PYTHON_USEDEP}]
+			dev-python/recommonmark[${PYTHON_USEDEP}]
+			dev-python/six[${PYTHON_USEDEP}]
+			dev-python/sphinx[${PYTHON_USEDEP}]
+			dev-python/sphinx-tabs[${PYTHON_USEDEP}]
+			dev-python/testresources[${PYTHON_USEDEP}]
+		')
+	)
+	opengl? (
+		media-libs/freeglut
+		media-libs/glew:=
+		media-libs/libglvnd
+	)
+"
+# 	test? (
+# 		>=media-libs/openimageio-2.2.14
+# 		>=media-libs/osl-1.11
+# 	)
+# "
+
+# Restricting tests, bugs #439790 and #447908
+RESTRICT="!test? ( test )"
```

Comment:
Is this still relevant? Can we fix them (at least a bit)?

Reply:
There are still multiple failures like this https://github.com/AcademySoftwareFoundation/OpenColorIO/issues/1361.

But since this has been disabled forever and the bump likely fixes https://bugs.gentoo.org/877753. It might as well stay for now.

Reply:
Wait, I just realised - they're.. not restricted, the comment just says they are?